packages/docs/PropTables/TablePropTable.tsx

```diff
@@ -1,45 +1,117 @@
+    types: (
+      <NextLink href="/Pagination/PaginationPage" as="/pagination">
+        Pagination
+      </NextLink>
+    ),
+    description: 'See pagination component for details.',
+  },
+  {
+    name: 'selectable',
+    types: <NextLink href="#table-selectable-prop-table">Selectable</NextLink>,
+    description: (
+      <>
+        See <NextLink href="#table-selectable-prop-table">below</NextLink> for usage.
+      </>
+    ),
+  },
+  {
+    name: 'stickyHeader',
+    types: 'boolean',
+    description: 'Makes the table header fixed.',
+  },
+];
 
-export const TablePropTable: React.FC = () => (
-  <PropTable>
-    <PropTable.Prop name="selectable" types="boolean">
-      Determines whether table rows are selectable or not.
-    </PropTable.Prop>
-    <PropTable.Prop name="stickyHeader" types="boolean">
-      Enables/disables the table header as sticky.
-    </PropTable.Prop>
-  </PropTable>
-);
+const tableColumnsProps: Prop[] = [
+  {
+    name: 'render',
+    types: 'React.ComponentType<Item>',
+    required: true,
+    description: 'Component used to render a column.',
+  },
+  {
+    name: 'header',
+    types: 'string',
+    required: true,
+    description: 'Header title.',
+  },
+  {
+    name: 'align',
+    types: ['left', 'center', 'right'],
+    defaultValue: 'left',
+    description: 'Sets alignment for column.',
+  },
+  {
+    name: 'verticalAlign',
+    types: ['top', 'center'],
+    defaultValue: 'top',
+    description: 'Sets vertical alignment for column (td only).',
+  },
+  {
+    name: 'width',
+    types: ['string', 'number'],
+    description: 'Sets column width.',
+  },
+  {
+    name: 'withPadding',
+    types: 'boolean',
+    defaultValue: true,
+    description: 'Toggles padding on td elements.',
+  },
+];
+
+const tableSelectableProps: Prop[] = [
+  {
+    name: 'selectedItems',
+    types: 'Item[]',
+    description: 'Defines which items are selected.',
+  },
+  {
+    name: 'onSelectionChange',
+    types: '(selectedItems: Item[]) => void',
+    description: 'Function to be called when item selection changes.',
+  },
+  {
+    name: 'itemType',
+    types: 'string',
+    description: 'Item type name displayed on the selected action section. (Eg: 2/10 Products)',
+  },
+];
 
-export const TableActionsPropTable: React.FC = () => (
-  <PropTable>
-    <PropTable.Prop name="tableId" types="string">
-      By default, <Code primary>tableId</Code> will be generated uniquely, otherwise pass in a user-defined id.
-    </PropTable.Prop>
-  </PropTable>
+export const TablePropTable: React.FC<PropTableWrapper> = props => (
+  <PropTable title="Table" propList={tableProps} {...props} />
 );
 
-export const TableBodyPropTable: React.FC = () => (
-  <Text>
-    Supports all native <Code>&lt;tbody&nbsp;/&gt;</Code> element attributes.
-  </Text>
+export const TableColumnsPropTable: React.FC<PropTableWrapper> = props => (
+  <PropTable title="Table[Columns]" propList={tableColumnsProps} {...props} />
```

Comment:
Not sure if we I'm 100% with using `Table[]`. Shouldn't `Columns` be enough?

Reply:
🤔yeah I'm not sure whats the best name for these, I do want to somehow how hierarchy tho, this way if `Columns` gets imported and rendered somewhere else it's pretty explicit it belongs to Table.